draft-ietf-ohai-svcb-config.md

```diff
@@ -346,11 +346,14 @@ For oblivious DoH servers, an attacker could use unique `dohpath` values
 to target or identify specific clients. This attack is very similar to
 the generic OHTTP key targeting attack described above.
 
-Clients SHOULD mitigate such attacks. This can be done with a
-check for consistency, such as using a mechanism described in {{CONSISTENCY}}
-to validate the `dohpath` value with another source. It can also be
-done by limiting the allowable values of `dohpath` to a single
-value, such as the commonly used "/dns-query{?dns}".
+A client can avoid these targeting attacks by only allowing a single
+`dohpath` value, such as the commonly used "/dns-query{?dns}" or
+another pre-known value. If the client allows arbitrary `dohpath`
+values, it SHOULD mitigate the attack with a consistency check,
+such as using a mechanism described in {{CONSISTENCY}} to validate the
+`dohpath` value with another source. Clients might choose to only
+employ a consistency check on a percentage of discovery events,
+depending on the capacity of consistency check options.
```

Comment:
```suggestion
depending on the capacity of consistency check options and their
deployment threat model.
```

Reply:
We may want to explain the scenarios in which the consistency check can be skipped by the client.

Reply:
I think providing an example would help.